src/bin/nydus-image/unpack.rs

```diff
@@ -0,0 +1,1112 @@
+use std::{
```

Comment:
As we already have directory `src/bin/nydus-image/unpack`, how about naming this file as `src/bin/nydus-image/unpack/mod.rs`?